api/webhook.go

```diff
@@ -402,14 +402,6 @@ func PostWebhook(c *gin.Context) {
 			if err != nil {
 				retErr := fmt.Errorf("%s: unable to get pipeline configuration for %s: %w", baseErr, r.GetFullName(), err)
 
-				// check if the retry limit has been exceeded
-				if i < retryLimit {
-					logrus.WithError(retErr).Warning("retrying")
-
-					// continue to the next iteration of the loop
-					continue
-				}
-
```

Comment:
opted to remove this retry logic since ConfigBackoff is already retrying 5x. if that fails it's likely this is just failing, period.